src/server.py

```diff
@@ -185,7 +185,10 @@ def sound_location(self):
         return random.choice(self.metadata["sound-files"])
 
     def _add_keyframe_pair(self, control, time_start_ns, value_start,
-                           time_end_ns, value_end, consider_duration=True):
+                           time_end_ns, value_end, consider_duration=True,
+                           unset_all=True):
```

Comment:
If `unset_all=False` is never used, then we don't really need to have it as an option. Just always unset the subsequent control points.